linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_retry/rule.yml

```diff
@@ -56,29 +56,42 @@ references:
     stigid@rhel8: RHEL-08-020104
     stigid@ubuntu2004: UBTU-20-010057
 
-ocil_clause: 'it is not the required value'
+ocil_clause: 'the value of "retry" is set to "0" or greater than "{{{ xccdf_value("var_password_pam_retry") }}}", or is missing'
 
 ocil: |-
-    To check how many retry attempts are permitted on a per-session basis, run the following command:
+    Verify {{{ full_name }}} is configured to limit the "pwquality" retry option to {{{ xccdf_value("var_password_pam_retry") }}}.
+
     {{% if product in ['rhel8', 'rhel9'] %}}
+    Check for the use of the "pwquality" retry option in the pwquality.conf file with the following command:
     <pre>$ grep retry /etc/security/pwquality.conf</pre>
     {{% else %}}
+    Check for the use of the "pwquality" retry option in the PAM files with the following command:
     {{% if 'ubuntu' in product %}}
     <pre>$ grep pam_pwquality /etc/pam.d/common-password</pre>
     {{% else %}}
     <pre>$ grep pam_pwquality /etc/pam.d/system-auth</pre>
     {{% endif %}}
+
+    <pre>password required pam_pwquality.so retry={{{ xccdf_value("var_password_pam_retry") }}}</pre>
```

Comment:
Note that the variable is not set explicitly in the srg_gpos control files. I think it is a good idea to do that.

Reply:
It is explicitly set in the `srg_gpos.yml` file. However, I agree it is not the most intuitive place and moved it to the `SRG-OS-000069-GPOS-00037.yml` file.